packages/swi_prolog.rb

```diff
@@ -0,0 +1,52 @@
+require 'package'
+
+class Swi_prolog < Package
+  description 'SWI-Prolog offers a comprehensive free Prolog environment. Since its start in 1987, SWI-Prolog development has been driven by the needs of real world applications. SWI-Prolog is widely used in research and education as well as commercial applications. Join over a million users who have downloaded SWI-Prolog.'
+  homepage 'https://www.swi-prolog.org/'
+  @_ver = '8.2.4'
+  version @_ver
+  compatibility 'x86_64', 'i686'
```

Comment:
I'm thinking this should be `'i686,x86_64'`?  Probably should change the logic to check for compatibility in crew anyway.